spec/feature/help_spec.rb

```diff
@@ -0,0 +1,14 @@
+RSpec.describe "Help", type: :feature do
+  context "rendering the help index page" do
+    before do
+      payload = { base_path: "/help", format: "special_route", title: "Help using GOV.UK", description: "" }
+      stub_content_store_has_item("/help", payload)
+    end
+
+    it "renders the help index page correctly" do
+      visit "/help"
+
+      expect(page).to have_css("h1", text: "Help using GOV.UK")
```

Comment:
Minor: `expect(page).to have_title("Help using GOV.UK")`

Reply:
Done. Thanks! that makes much more sense in the tests.